core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,66 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
+      UPDATE
+        spree_adjustments
+      SET
+        order_id = adjustable_id
+      WHERE
+        adjustable_type = 'Spree::Order'
+      ;
+    SQL
+
+    # Whoever runs a big enough MySQL installation where the AR solution hurts:
+    # Will have to write a better MySQL specific equivalent.
+    if Spree::Order.connection.adapter_name.eql?('MySQL')
```

Comment:
Not a big deal at all but I believe you can just do `connection.adapter_name...` here